Packs/CTIX/Integrations/CTIXv3/CTIXv3.py

```diff
@@ -74,6 +84,29 @@
     "url": "url",
 }
 
+THREAT_DATA_MAP = {
```

Comment:
remove this const
you don't use it in the code at all

Reply:
Fixed.